awslogs.conf

```diff
@@ -2,8 +2,15 @@
 state_file = /var/log/awslogs-state
 
 [application-logs]
-file = /var/log/application.log
+file = /var/log/digitalmarketplace/application.log
 log_group_name = ${DM_ENVIRONMENT}-${DM_APP_NAME}
-log_stream_name = {hostname}
-datetime_format = "%Y-%m-%dT%H:%M:%S"
+log_stream_name = Application-{hostname}
```

Comment:
How do we use the logs once they're in the log group? Will they get exported to Kibana in some way? It seemed like a neater solution to have each apps logs in one group.